docs/security-policy-for-authors.md

```diff
@@ -0,0 +1,181 @@
+
+   At the bare minimum, it should provide a current package
+   maintainer's email address (or ideally an alias that forwards to
+   multiple authors), or a bug reporting system that supports special
+   handling of security issues.
+
+   Some examples,
+
+   > Security vulnerabilties can be reported by e-mail to the current
+   > project maintainers at <fobar@example.com>.
+   >
+   > Please include as may details, including code samples, so that we
+   > can reproduce the issue.
+
+   or [1]
+
+   > Security vulnerabilities can be reported via our GitHub
+   > repository at https://example.github.com/foobar. On the
+   > "Advisories" page you can click on the "Report a vulnerability"
+   > button.
+
+   Add note about also copying cpan-security on the notification if
+   the security issue is a 0-day vulnerability or is otherwise urgent.
+
+   For example
+
+   > If this is a 0-day vulnerability that is actively being
+   > exploited, then please copy the report to the CPAN Security Group
+   > at <cpan-security@perl.org>.
+
+   Advise users on what to expect in terms of a response.  If there is
+   a formal vulnerability workflow than refer to that section of the
+   document.
+
+   A recommendation that after a certain amount of time not hearing
+   from the author(s), to report to cpan-security@perl.org.
+
+   For example
+
+   > We will aim to acknowledge to your security report within 72 hours.
+   > However, this project is maintained by a small group of
+   > volunteers.  If you have not heard back from the maintainers
+   > within 72 hours (three days), then please forward the report to
+   > the CPAN Security Group at <cpan-security@perl.org>.
+   >
+   > Please note that the initial response to your report will be an
+   > acknowledgement, with a possible query for more information. It
+   > will not necessarily include any fixes for the issue.
+
+   Also note on what conditions you may forward the security issue to
+   other people. For example,
+
+   > The project maintainers may forward this issue to the security
+   > contacts for other projects where we believe it is relevant to
+   > them. This may include embedded libraries or prerequisite modules
+   > or system libraries, or downstream software that uses this
+   > software.
+   >
+   > We may also forward this issue to the CPAN Security Group.
+
+   It is important to advise users not to report security
+   vulnerabilities in public, e.g.
+
+   > Please *do not* use the public issue reporting system on RT for
+   > reporting security vulnerabilities.
+   >
+   > Please do not disclose the security vulnerability in public. That
+   > includes blogs, social media, forums, or conferences.
+
+   Note: many modules include a "BUGS" section in their POD that
+   describes how to report a bug.  These sections should include a
+   subsection that advises how to safely report security issues, and
+   should refer readers to the `SECURITY` document.
+
```

Comment:
I am wondering if it's worth mentioning PGP keys as on option?

Note PGP encrypted messages are problematic if there are multiple maintainers that need to be notified.

Reply:
maybe this is a topic suitable to discuss during the next CPANSec meeting?